spacy/cli/package.py

```diff
@@ -113,8 +114,18 @@ def package(
             msg.fail(
                 f"Model name ('{name}') is not a valid python identifier. "
                 "This is required to correctly package the pipeline so it can be imported as a module.",
```

Comment:
I'm still not very fond of "python identifier" here. Something more like "not a valid module name" and "so it can be imported as a module" and not about packaging per se. I think the link is going to be confusing given the next check, since it doesn't really describe the final constraints.